dev-infrastructure/Makefile

```diff
@@ -22,7 +22,14 @@ MGMG_RG_DEPLOYMENT_NAME ?= mgmt${DEPLOYMENT_NAME_SUFFIX}
 SVC_RG_DEPLOYMENT_NAME ?= svc${DEPLOYMENT_NAME_SUFFIX}
 REGIONAL_RG_DEPLOYMENT_NAME ?= region${DEPLOYMENT_NAME_SUFFIX}
 ROLE_ASSIGNMENTS_DEPLOYMENT_NAME ?= role-assignments${DEPLOYMENT_NAME_SUFFIX}
-AUTOMATION_ACCOUNT_RG = "hcp-dev-automation-account"
+
+# Subscription IDs
+DEV_SUBSCRIPTION_ID = 1d3378d3-5a3f-4712-85a1-2485495dfc4b
+INT_SUBSCRIPTION_ID = 64f0619f-ebc2-4156-9d91-c4c781de7e54
+STAGE_SUBSCRIPTION_ID = b23756f7-4594-40a3-980f-10bb6168fc20
```

Comment:
Ev2RA injects the subscription for you - why do you need to track IDs for prod?

Reply:
- `int` refers to the [ARO SRE Team - INT (EA Subscription 3)](https://portal.azure.com/#@redhat0.onmicrosoft.com/resource/subscriptions/64f0619f-ebc2-4156-9d91-c4c781de7e54/overview)
- `stage` refers to the [ARO HCP - STAGE testing (EA Subscription)](https://portal.azure.com/#@redhat0.onmicrosoft.com/resource/subscriptions/b23756f7-4594-40a3-980f-10bb6168fc20/overview)

Any suggestion for better names?